.github/workflows/build.yml

```diff
@@ -26,10 +26,29 @@ jobs:
       run: |
         sudo apt-get -y install pandoc
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 10
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
+    # We allow at most 5 commits in a branch to ensure our CI doesn't break.
+    - name: Check commit count in PR
+      if: always()
+      shell: bash
+      run: |
+        set -x 
+        git fetch origin $GITHUB_REF:commit-count
+        git fetch origin master
+        diff=$(git rev-list --count origin/master...commit-count)
+        # $GITHUB_REF adds an additional commit to the commit tree, so $diff is
```

Comment:
what is $GITHUB_REF?

Reply:
(maybe put a general comment explaining what GITHUB_REF is right after the `set -x` line?)

Reply:
Added a comment.